internal/controllers/core/cmd/execer.go

```diff
@@ -150,14 +150,23 @@ func (e *processExecer) processRun(ctx context.Context, cmd model.Cmd, w io.Writ
 	defer close(statusCh)
 
 	logger.Get(ctx).Infof("Running cmd: %s", cmd.String())
-	c := e.localEnv.ExecCmd(cmd, logger.Get(ctx))
+	c, err := e.localEnv.ExecCmd(cmd, logger.Get(ctx))
+	if err != nil {
+		logger.Get(ctx).Errorf("%s invalid cmd: %v", cmd.String(), err)
```

Comment:
```suggestion
		logger.Get(ctx).Errorf("%q invalid cmd: %v", cmd.String(), err)
```

i usually quote the comment with %q, so it's easier to see how whitespace plays in